rest/route/service.go

```diff
@@ -258,7 +259,6 @@ func AttachHandler(app *gimlet.APIApp, opts HandlerOpts) {
 	app.AddRoute("/users/{user_id}/permissions").Version(2).Post().Wrap(requireUser, editRoles).RouteHandler(makeModifyUserPermissions(env.RoleManager()))
 	app.AddRoute("/users/{user_id}/permissions").Version(2).Delete().Wrap(requireUser, editRoles).RouteHandler(makeDeleteUserPermissions(env.RoleManager()))
 	app.AddRoute("/users/{user_id}/roles").Version(2).Post().Wrap(requireUser, editRoles).RouteHandler(makeModifyUserRoles(env.RoleManager()))
-	app.AddRoute("/users/permissions").Version(2).Get().Wrap(requireUser).RouteHandler(makeGetAllUsersPermissions(env.RoleManager()))
```

Comment:
Alternatively, we could move this route above /users/{user_id} instead. No other routes have this issue (e.g. the /users/offboard_user and /users/rename_user) because they are post requests while this is a get request. 

But I think it makes sense to move like you did because the other users route deal with one user while this deals with many users

Reply:
I thought about that, but I do just prefer this naming scheme.